Core/clim-core/panes/composition.lisp

```diff
@@ -201,6 +201,14 @@
   ()
   (:documentation "protocol class"))
 
+;; When a composite-pane receives the note-space-requirements-changed
+;; from one of its children, the normal behaviour is to change its
+;; space-requirements. For the moment exception are restraining-pane
+;; and viewport-pane. -- admich 2020-08-11
+(defmethod note-space-requirements-changed  ((pane composite-pane) client)
+  (declare (ignore client))
+  (change-space-requirements pane))
+
```

Comment:
I suppose that i.e vertical pane should call compose-space on the child and if its new space requirements may be fulfilled by the current size, then change-space-requirements doesn't need to be called. Is that right?

Reply:
I think you are right and that can be implemented.

Reply:
I'll treat it as a possible improvement.